src/profile-logic/profile-data.js

```diff
@@ -914,9 +914,16 @@ export function getTracingMarkers(thread: Thread): TracingMarker[] {
   for (let i = 0; i < markers.length; i++) {
     const data = markers.data[i];
     if (!data) {
-      continue;
-    }
-    if (data.type === 'tracing') {
+      // Add a marker with a zero duration
+      const marker = {
+        start: markers.time[i],
+        dur: 0,
+        name: stringTable.getString(markers.name[i]),
+        title: null,
+        data: null,
+      };
+      tracingMarkers.push(marker);
```

Comment:
> I feel like a null duration rather than 0 

Except we can have interval markers with a duration 0 too (exemple: UserTiming of type "mark") :) so I think it's better to check for a 0 duration instead.